docs/community/governance.md

```diff
@@ -82,7 +82,7 @@ After the RFC has been discussed, a unanimous vote will be required for the new
 
 ## Repository Maintainers
 
-Repository Maintainers are trusted stewards of the PowerShell repository responsible for maintaining consistency and quality of PowerShell code.
+Repository Maintainers are trusted users of the PowerShell community responsible for maintaining consistency and quality of PowerShell code/repository.
```

Comment:
I don't understand the clarification here.  Changing from stewards to users is a significant change in the meaning here (triggering the committee review.)   
```suggestion
Repository Maintainers are trusted stewards of the PowerShell community/repository responsible for maintaining consistency and quality of PowerShell code.
```

I suggest this change to keep the original meaning.